dropfile_formats/DOOR.SYS

```diff
@@ -0,0 +1,52 @@
+COM1:
```

Comment:
What do you think about supporting comments in the templates so we can self-doc? I think `// anything after` would work for all of them